samples/react-native/ios/Podfile

```diff
@@ -62,7 +47,8 @@ target 'sentryreactnativesample' do
     react_native_post_install(
       installer,
       config[:reactNativePath],
-      :mac_catalyst_enabled => false
+      :mac_catalyst_enabled => false,
+      # :ccache_enabled => true
```

Comment:
Q: Should this be enabled?

Reply:
I think we didn't have cache enabled before. I brought the comment over for alignment with [the RN 0.75 sample](https://react-native-community.github.io/upgrade-helper/?from=0.73.9&to=0.75.4#RnDiffApp-ios-Podfile).

Reply:
We can explore the cache in the future. We didn't have CCache enabled before.

Reply:
I've created an issue to revisit this https://github.com/getsentry/sentry-react-native/issues/4170